lib/src/data/simulations.dart

```diff
@@ -63,84 +65,94 @@ class Simulations with ChangeNotifier {
       ),
       SimulationCard(
         id: 1,
+        simulationName: 'Selection Sort',
+        image: theme.darkTheme
+            ? 'assets/simulations/SelectionSortDark.png'
+            : 'assets/simulations/SelectionSortLight.png',
```

Comment:
I believe we can re-use the `InsertionSort` image assets and we can remove the new ones.